test/plain-list/AutosuggestApp.test.js

```diff
@@ -456,6 +803,19 @@ describe('Plain list Autosuggest', () => {
       });
     });
 
+    it('should not call onChange when Enter is pressed and suggestion is focused with down since value did not change', () => {
+      onChange.reset();
+      clickDown();
+      clickEnter();
+
+      // onChange should not have been called because the value didn't change
+      expect(onChange).to.have.been.calledExactlyOnce;
+      expect(onChange).to.be.calledWithExactly(eventInstance, {
+        newValue: 'Java',
+        method: 'down'
+      });
+    });
```

Comment:
1. The description says "should not call onChange", but we expect it to be called once. What am I missing here?
2. If we exercise `onChange` behaviour, let's put the test in the `describe('inputProps.onChange', ...)` section